README.md

```diff
@@ -215,8 +198,14 @@ the FV3 makefiles:
 
 And build the model
 
-    cd FV3
-    make
+    make -C FV3
+
+Then, to run some simple tests execute
+
+    pytest --native tests/pytest
+
+The `--native` flag only runs tests intended to be run in a native environment
```

Comment:
it might be helpful in the 'testing the mode' section below to also include a description of what gets tested how with and without the --native flag.

Reply:
Thanks. I moved this documentation to that section and left a cross-reference in place. 9df5bfa2b162fac9baf46fcfeb0787c267f1d4bc